### PR TITLE
Migrate OperationsMenuController to kotlin

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Signed-off-by: Andy Scherzinger <info@andy-scherzinger.de>

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+@AutoInjector(NextcloudTalkApplication::class)
+class OperationsMenuController(args: Bundle) : NewBaseController(
+    R.layout.controller_operations_menu,
+    args
+) {
+    private val binding: ControllerOperationsMenuBinding by viewBinding(ControllerOperationsMenuBinding::bind)
+
+    @JvmField
+    @Inject
+    var ncApi: NcApi? = null
```
Comment (timkrueger) [inline app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt:92]: Can this be changed to the following?

```java
    @Inject
    lateinit var ncApi: NcApi
```

File: detekt.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 build:
-  maxIssues: 95
+  maxIssues: 100
```
Comment (timkrueger) [inline detekt.yml:2]: I will try to get the `maxIssues` back to 95.

**Review by timkrueger — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt (hunk 1 above)
Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt:92]: Yeah, makes sense 👍

Comment (timkrueger) [reply app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt:92]: I can do if you not already did.

**Review by timkrueger — APPROVED** [review]

**Pull request merged**